src/main/java/org/folio/circulation/domain/notice/schedule/ScheduledNoticesRepository.java

```diff
@@ -43,4 +52,42 @@ public ScheduledNoticesRepository(
       }
     });
   }
+
+  public CompletableFuture<Result<MultipleRecords<ScheduledNotice>>> findNoticesToSend(
+    DateTime systemTime, int pageLimit) {
+    return CqlQuery.lessThan("nextRunTime", systemTime.withZone(DateTimeZone.UTC))
```

Comment:
Is the less than operator implemented in the fake CQL implementation. If not, this might not behave as expected during tests.

Are there API tests for demonstrating that the scheduled API does not pick up notices in the future?